src/overlays/actors/ovl_En_Nb/z_en_nb.h

```diff
@@ -6,11 +6,84 @@
 
 struct EnNb;
 
+typedef void (*EnNbActionFunc)(struct EnNb*, GlobalContext*);
+typedef void (*EnNbDrawFunc)(struct EnNb*, GlobalContext*);
+
 typedef struct EnNb {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1DC];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ Vec3s limbDrawTable[19];
+    /* 0x0202 */ Vec3s transitionDrawTable[19];
+    /* 0x0274 */ s16 eyeIdx;
+    /* 0x0276 */ s16 blinkTimer;
+    /* 0x0278 */ s32 action;
```

Comment:
this is an action function index right, not the cutscene actor action?

Reply:
Yes

Reply:
maybe name it `actionFuncIndex` or something then? idk

or actionMode for consistency with drawMode (which I would otherwise rename to drawFuncIndex), but I don't think "mode" is nice, as it'd be a single use in the whole codebase?